data/boarding missions.txt

```diff
@@ -34,6 +146,106 @@ mission "Assisting Hai"
 	on offer
 		payment 30000
 		conversation
-			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
+			`Knowing that you endangered your life to save the <origin>, the captain grants you <payment>.`
 				decline
 	destination Hai-home
+	
+mission "Assisting Unfettered Hai (Thief)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment -300000
+		conversation
+			`After assisting the <origin> you find that 300000 credits have been removed from your account.`
+				decline
+	destination Darkcloak
+	
+mission "Assisting Unfettered Hai(redo)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment 20000
+		conversation
+			`After assisting the <origin> you find that <payment> credits have been added to your account.`
+				decline
+	destination Darkcloak
+	
+mission "Assisting Wanderers (Pre-Language)"
```

Comment:
It might not make sense to have Wanderers pay you for assisting them pre-language, let alone having them pay you substantially less pre-language.